pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[project]
+name = "apis"
+version = "0.25.0"
+description = "Attendee registration software for fan conventions"
+readme = "README.md"
+requires-python = ">=3.11"
```

Comment:
The base image still uses 3.10 - do we want to bump the base image to 3.11 or lower this to 3.10?

Reply:
id say bump to 3.11 probably

Reply:
I can take care of that tomorrow if you'd like